src/components/shared/sider/sider-body.tsx

```diff
@@ -160,16 +178,33 @@ const SiderBody = () => {
 						<Autocomplete
 							id="realm choice"
 							options={
-								realms?.map(
-									(realm) => realm.name,
+								realms?.map((realm) =>
+									realm.name ===
+									favoriteRealm
+										? realm.name + ' ☆'
+										: realm.name,
 								) || []
 							}
 							style={{ width: 300 }}
-							value={realmSelected?.name || null}
+							value={
+								(realmSelected?.name &&
+								realmSelected?.name ===
+									favoriteRealm
+									? realmSelected?.name +
+									  ' ☆'
+									: realmSelected?.name) ||
+								null
+							}
 							onChange={(
 								_event: any,
 								newRealmName: string | null,
 							) => {
+								newRealmName =
```

Comment:
maybe you can separate the value rendered from the real value to avoid this kind of problem

Reply:
maybe you can try this : 
`							getOptionLabel={(option) =>
								option === favoriteRealm
									? option + ' ☆'
									: option
							}`